### ISSUE TITLE
Can we use `subscribe` in bus protocol?

### ISSUE THREAD

**Issue by sailxjx** [nanomsg/nng#1568]
I use bus protocol to construct a mesh of services, however, not every service is interested in all topics. So, to reduce the network overhead, is there a way to support `subscribe(topic)` in bus protocol?

```
with pynng.Bus0() as sock:
        sock.listen(listen_address)
        for contact in contacts:
            sock.subscribe("topic")  # Need this api
            sock.dial(contact)
```

**Comment by gdamore** [comment]
No. And furthermore, sub protocol doesn't reduce network traffic either.

If you need to do this you might want to use separate sockets for different purposes/topics.
